recipe/meta.yaml

```
@@ -58,6 +58,12 @@ requirements:
     - llvmdev {{ llvm_version }}
     - aws-crt-cpp
     - aws-sdk-cpp
+    # azure filesystem dependencies
+    - azure-core-cpp
+    - azure-identity-cpp
+    - azure-storage-blobs-cpp
+    - azure-storage-common-cpp
+    - azure-storage-files-datalake-cpp
```

and restrict the dependencies here
```suggestion
    - azure-core-cpp                    # [unix]
    - azure-identity-cpp                # [unix]
    - azure-storage-blobs-cpp           # [unix]
    - azure-storage-common-cpp          # [unix]
    - azure-storage-files-datalake-cpp  # [unix]
```